src/bullet/index.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components'
+import { color } from '_utils/branding'
+import Bullet from './Bullet'
+
+const StyledBullet = styled(Bullet)`
+  & {
+    box-sizing: content-box;
+    width: 6px;
+    height: 6px;
+    background-color: ${color.white};
+    border: 2px solid ${color.primaryText};
+    border-radius: 50%;
+  }
+
+  &.kirk-bullet--addon {
+    border-color: ${color.border};
```

Comment:
Not sure we want to use the border semantic here.

Reply:
Yes, I only moved what was already there. Would `color.fadedText` be OK? It's actually used along with an actual faded text